aspnetcore/fundamentals/servers/kestrel.md

```diff
@@ -130,6 +132,21 @@ The following examples use the <xref:Microsoft.AspNetCore.Server.Kestrel.Core> n
 using Microsoft.AspNetCore.Server.Kestrel.Core;
 ```
 
+Kestrel options, which are configured in C# code in the following examples, can also be set using a [configuration provider](xref:fundamentals/configuration/index). For example, the File Configuration Provider can load Kestrel configuration from a file (*appsettings.json* or *appsettings.{Environment}.json*) when `CreateDefaultBuilder` is called:
+
+```json
+{
+  "Kestrel": {
+    "Limits": {
+      "MaxConcurrentConnections": 100,
+      "MaxConcurrentUpgradedConnections": 100
+    },
+    "AllowSynchronousIO": true,
```

Comment:
```suggestion
    "AllowSynchronousIO": false,
```
True is strongly discouraged.

Reply:
I'll just strip that ... it's not needed to make the point.